### PR TITLE
Add kernel bin, minivm, Enable slirp for QEMU, update to 19.1.0

### PR DESCRIPTION
**Pull request by androm3da**

### CODE REVIEW

File: build-toolchain.sh [hunk 1]
```diff
@@ -239,6 +239,7 @@ build_qemu() {
 	CC=$(which gcc) \
 	PATH=${TOOLCHAIN_BIN}:${PATH} \
 	../qemu/configure --enable-fdt --disable-capstone --disable-guest-agent \
+	                  --enable-slirp \
```
Comment (quic-mathbern) [inline build-toolchain.sh:242]: We use ubuntu 20 at codelinaro CI, right? If so, OK, libslirp should be available

File: build-rootfs.sh [hunk 2]
```diff
@@ -115,6 +115,16 @@ build_dropbear() {
 	echo FIXME TODO
 }
 
+build_minivm() {
+	cd ${BASE}
+	mkdir -p obj_minivm
+	cd obj_minivm
+	make --directory=../hexagonMVM \
+		ARCH=hexagon-unknown-linux-musl- \
+		DESTDIR=${ROOTFS}/boot \
```
Comment (quic-mathbern) [inline build-rootfs.sh:124]: Ok, I see, this depends on https://github.com/quic/hexagonMVM/pull/10/files

**Review by quic-mathbern — APPROVED** [review]

File: build-toolchain.sh (hunk 1 above)
Comment (androm3da) [reply build-toolchain.sh:242]: Good question - Looks like it should come via the `apt build-dep ... qemu ...` in the container (the one that debian/ubuntu ship is pretty full-featured so we should get a Whole Bunch of optional dependencies like this one satisfied).

Comment (quic-mathbern) [reply build-toolchain.sh:242]: Ah, ok, nice

File: build-rootfs.sh (hunk 2 above)
Comment (androm3da) [reply build-rootfs.sh]: Yup and it will use the `v0.2.0` from hexagonMVM once it's ready.

**Pull request merged**